wemake_python_styleguide/logic/scopes.py

```diff
@@ -145,46 +149,37 @@ def is_function_overload(node: ast.AST) -> bool:
     return False
 
 
-def _is_var_imported_from(
-    node: ast.AST,
-) -> bool:
-    """Check that name imported from module."""
-    ctx = get_context(node)
-    if ctx and isinstance(node, ast.Name):
-        for part in ctx.body:
-            if isinstance(part, ast.ImportFrom):
-                imported_names = {
-                    imported.asname
-                    if imported.asname else imported.name
-                    for imported in part.names
-                }
-                if node.id in imported_names:
-                    return True
-    return False
-
-
-def _is_tuple_have_imported_name(node: ast.AST) -> bool:
-    if isinstance(node, ast.Tuple):
-        assigned_vars = [
-            el.value
-            if isinstance(el, ast.Starred)
-            else el
-            for el in node.elts
-        ]
-        return any(
-            (_is_var_imported_from(assigned) for assigned in assigned_vars),
-        )
-    return False
+def _get_variables_from_complex_node(node: ast.AST) -> Set[str]:
+    names = set(get_variables_from_node(node))
+    if isinstance(node, ast.Attribute):
```

Comment:
We should check for just `ast.Name` here. Are there any other cases that are not covered with `ast.Name`?